File: backend/src/main/java/br/com/brunodezorzi/API/noticiario/dto/CategoriaDTO.java
package br.com.brunodezorzi.api.noticiario.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class CategoriaDTO {

    private String nome;

    private String descricao;

    private Boolean  destaque;

    private Integer categoriaPaiId;
    







}

File: frontend/src/app/dto/categoria-dto.ts
export class CategoriaDTO {
    nome!: string;
    descricao!: string;
    categoriaPaiId?: number | null;

    constructor(nome?: string, descricao?: string, categoriaPaiId?: number | null) {
        this.nome = nome || '';
        this.descricao = descricao || '';
        this.categoriaPaiId = categoriaPaiId || null;
    }
}